### ISSUE TITLE
Lista de símbolos / List of symbols

### ISSUE THREAD

**Issue by GoogleCodeExporter** [abntex/abntex2#115]
```
Which component is your suggestion addressed to?
[ ] class
[ ] packages
[ ] bibliography style
[ ] documentation
[ ] examples
[x] other: Lista de símbolos / List of symbols
What is you suggestion?

Pessoal, antes de tudo, acho ótimo o trabalho de vocês com o abntex2.

Bem, ainda não temos uma 'lista de símbolos' que funcione como a 'lista de 
figuras', por exemplo. Eu venho tentando obter essa funcionalidade e estou 
quase finalizando. Por isso, peço a ajuda de vocês: com o código abaixo eu 
consigo a aparência desejada, mas, apesar de serem mostrados o números 
corretos, os links enviam para a página errada no documento PDF final. O que 
pode estar errado?

---

Guys, first of all, you're doing great with the abntex2.

Well, we don't have yet a 'list of symbols' that works like a 'list of 
figures', e.g. I have been trying to achieve this functionality and I'm almost 
there. So I ask your help: with the code below I have the look I desire but, 
despite it shows the correct number the links sends you a wrong page on the 
final PDF document. What can be wrong?

---

Additional information: Code

% ***** Definicao / Definition *****
% Escreve o simbolo no texto e uma entrada na Lista de Simbolos.
% Writes the symbol on text and an entry in symbol list
% Uso/Usage:
% \simb[entrada na lista de simbolos]{simbolo}
% \simb[caption in symbol list]{symbol}

\newcommand{\simb}[2][]{%
    \ifthenelse{\equal{#1}{}}
        {\addcontentsline{los}{simbolo}{\hbox to 21pt{\ensuremath{#2}}}}
        {\addcontentsline{los}{simbolo}{\hbox to 21pt{\ensuremath{#2}}
        \hspace*{6pt}{#1}}}
        \ensuremath{#2}
}
\makeatletter
% \listadesimbolos: imprime a lista de simbolos / prints the symbol list
\newcommand{\listadesimbolos}{
    \pretextualchapter{Lista de S\'imbolos}  % nome / name
    {\SingleSpacing\setlength{\parindent}{0cm}
    \@starttoc{los}}
}
\newcommand\l@simbolo[2]{\normalsize #1 \hspace*{2.5pt}\mydots #2 \par}

```

Original issue reported on code.google.com by `leobr...@gmail.com` on 2 May 2014 at 6:01
- Blocked on: #66, #104
- Merged into: #104

Attachments:
- [list.png](https://storage.googleapis.com/google-code-attachments/abntex2/issue-115/comment-0/list.png)

**Comment by Caroltcs** [comment]
#104
